packages/react/scripts/generateIcons.js

```diff
@@ -8,6 +8,7 @@ const iconNames = [];
 
 const template = ({ iconName, source }) => {
   return `import classNames from 'classnames';
+  import * as React from 'react';
```

Comment:
The old transform will need React in scope so that why I added it in a few places